packages/core/src/utils.ts

```diff
@@ -647,8 +600,8 @@ export const getNetworkTag = (
   ) {
     return networkName
   } else if (
-    Object.keys(SUPPORTED_NETWORKS).includes(networkName) &&
-    !Object.keys(SUPPORTED_LOCAL_NETWORKS).includes(networkName)
+    SPHINX_NETWORKS.find((n) => n.name === networkName) &&
+    !SPHINX_LOCAL_NETWORKS.find((n) => n.name === networkName)
```

Comment:
```suggestion
    !SPHINX_LOCAL_NETWORKS.some((n) => n.name === networkName)
```

nit: I think [Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) is more appropriate here, but `find` works too